docs/config.md

```diff
@@ -78,16 +78,16 @@ profile_models: <comma-delimited-string-list>
 [ override_output_model_repository: <boolean> | default: false ]
 
 # Concurrency values to be used for the analysis
-[ concurrency: <comma-delimited-string|list|range> | default: 1 ]
```

Comment:
It wouldn't be handled in the CLI code, but technically the true default if unspecified is 1,2,4,8,...,1024  (right @nv-braf ? Think we should mention the true default?)

Reply:
I just wanted to mention that the default value (before the refactor) should've been 1. The exponential increase that I mentioned during the meeting was only for the automatic sweep.

Reply:
ok, I'm fine to keep the doc how it is then.